### ISSUE TITLE
Cyclops and A new Creature

### ISSUE THREAD

**Issue by TheDinoguy** [AlexModGuy/Ice-and-Fire-Suggestions#490]
Cyclopses are one eyed giants born from the greek gods. In Jason and the Argonauts, the Cyclops Polyphemus attacked them and put his friends in a cave. In the myth different versions say that the cyclops threw a boulder at the boat. I think the Cyclops should have two new attacks. I couldn't help but be a little underwhelmed by the great monster. I would love to see a Area of Effect attack where it stomps the ground and makes a tiny earthquake that spans 8 blocks in each direction and I'd also love to see the Cyclops be able to pick up a boulder and hurl it at the player as a ranged attack. It doesn't have to be coded in that the Cyclops reaches into the ground to pick up a bunch of blocks and throw it at you but you could just make an animation where it burrows into the ground, roars, then hurls the boulder at the player.

Second Idea. I'd love to see Treants. Now I know lycanites mobs has them and so does Mo'Creatures but I think the Treants in this mod could be redone in a unique way that serves a purpose and gives a motivation to get players to seek out this creature. Treants can be neutral humanoid trees that stand tall in the forest. They walk around phasing through leaf blocks. Treants have an effect called Grace of the Earth where any saplings or crops grow faster. Treants can be tamed if you give them atleast more than 30 pieces of Bonemeal. Treants can wander near crops and cause them to grow faster. Alternatively if one is slain they drop the Treant twigs. Their twigs can be used to create a legendary staff that can add the regen potion effect or perform a thornwall attack. The staff can also be put on an altar in the middle of a crop field and cause any saplings or crops to grow quicker within a 40 block radius (radius can be configurable). Also there could be two skins for males and females Treants and Dryads. Their attacks include a ground pound attack, a stomp attack that sprouts thorny roots for a few seconds. A thornvine whiplash. They can also turn wolves hostile if they are in distress. They can spawn in Oak, Birch, Dark Oak, Taiga, and Jungle forest biomes with a skin for each wood type. Also if they are in a snow biome and it starts snowing they get a snowy texture on their branches.

So to recap Treants can be used as a guard mob or a AOE growth beacon for crops. They can also be used to make a new weapon like all the other mobs.

**Comment by Drakeofdark** [comment]
Capital idea
